dataeng/resources/tableau-backup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+
+ssh -tt centos@$TABLEAU_SERVER_IP -o StrictHostKeyChecking=no "
```

Comment:
I would recommend parameterizing the username as well, as we probably will be moving off of CentOS